frontend/src/components/groupPanel/groupPanel.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <div class="container">
+    <h1 class="title" id="title">Koło naukowe X</h1>
+    <p class="p-category">Informatyka</p>
+    <br />
+    <div class="columns">
+      <div class="box column is-3">
+        <b-button id="btn" size="is-medium" tag="router-link" to="/forum"
+          >Forum</b-button
+        >
+        <br />
+        <b-button id="btn" size="is-medium" v-on:click="showAboutUs"
+          >O nas</b-button
+        >
+        <b-button id="btn" size="is-medium" v-on:click="showWhatWeDo"
+          >Czym się zajmujemy</b-button
+        >
+        <b-button id="btn" size="is-medium" v-on:click="showMembers"
+          >Członkowie</b-button
+        >
+        <b-button id="btn" size="is-medium" v-on:click="showContact"
+          >Kontakt</b-button
+        >
+        <b-button
+          id="btn"
+          size="is-medium"
+          tag="router-link"
+          to="/group-tutorials"
+          >Materiały dydaktyczne</b-button
+        >
+      </div>
+      <div class="box column is-6" id="centerDiv">
+        <groupPanelTab
+          :title="selectedTabTitle"
+          :content="selectedTabContent"
+          v-if="selectedTabTitle != 'Kontakt'"
+        />
+        <groupPanelTabContact
+          :title="selectedTabTitle"
+          :content="selectedTabContent"
+          v-else
+        />
+      </div>
+      <div class="box column is-5" id="divLinks">
+        <b-menu :activable="false" :accordion="false" id="menu">
+          <b-menu-list>
+            <b-menu-item label="Linki">
+              <b-menu-item
+                label="Github"
+                target="_blank"
+                href="https://github.com"
+              ></b-menu-item>
+              <b-menu-item
+                label="Facebook"
+                target="_blank"
+                href="https://facebook.com"
+              ></b-menu-item>
+              <b-menu-item
+                label="Discord"
+                target="_blank"
+                href="https://discord.com"
+              ></b-menu-item>
+            </b-menu-item>
+            <b-menu-item label="Dyski">
+              <b-menu-item
+                label="Dysk 1"
+                target="_blank"
+                href="https://drive.google.com/drive/folders/1QMHnaSuOPcfOX16190I9D8q_Fa5pzeOF?usp=sharing"
+              ></b-menu-item>
+              <b-menu-item
+                label="Dysk 2"
+                target="_blank"
+                href="https://drive.google.com/drive/folders/1vGf8f0nVkJcAZQ6S7mZ74vFhpJK0Gbeo?usp=sharing"
```

Comment:
To będzię hardcodowane czy może będziemy im dawać możliowść dodania tego?

Reply:
będzie możliwość dodawania nowych linków i nowych dysków